request.js

```diff
@@ -343,23 +381,13 @@ Request.prototype.init = function (options) {
   if (options.formData) {
     var formData = options.formData
     var requestForm = self.form()
-    var appendFormValue = function (key, value) {
-      if (value && value.hasOwnProperty('value') && value.hasOwnProperty('options')) {
-        requestForm.append(key, value.value, value.options)
+    for (var i = 0, ii = formData.length; i < ii; i++) {
+      var formParam = formData[i]
+      if (!formParam) { continue }
+      if (formParam.options) {
```

Comment:
This should be handled in the `append` function.

Reply:
Sorry. Resolved by mistake. But, point is that the append function is in another library and not in our control.